helpers/auth_x11.c

```diff
@@ -1174,6 +1174,7 @@ int main() {
 #endif
 
   XSetWindowBackground(display, window, Background);
+  XMapRaised(display, window);
```

Comment:
(No change needed here, just discussing)
This works - I'd kinda prefer if the auth child _created_ the window instead as a child of the given window (and then we could pass down the saver window as usual), but this works too.

OTOH we could even explicitly allow that even with this protoocol, as the window given could also stay unmapped and be used by the auth child to locate its parent and create more children, and then be otherwise used as an event passing interface only (thinking of your infobar).

Reply:
That's a great way to solve the problem that I was starting to run into with the infobar (and also multiple lock screen windows: 1 per monitor).

I'm inclined to keep this as-is since it's relatively simple for the auth_x11 case, and have the new style unlock screen keep this window unmapped, using it only for communicating with main.